parser/prism/Translator.cc

```diff
@@ -1275,6 +1275,14 @@ unique_ptr<parser::Node> Translator::patternTranslate(pm_node_t *node) {
 
             return make_unique<parser::ArrayPattern>(location, move(sorbetElements));
         }
+        case PM_CAPTURE_PATTERN_NODE: { // A variable capture such as the `Integer => i` in `in Integer => i`
+            auto capturePatternNode = down_cast<pm_capture_pattern_node>(node);
+
+            auto value = translate(capturePatternNode->value);
```

Comment:
Maybe "pattern" is a better name, even if it doesn't match Prism

Reply:
Not saying the change is needed, but I generally lean towards expressing intent than matching naming from Prism.